tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java

```diff
@@ -137,37 +138,26 @@ private JsonElement parseValue(byte type, DataInput di) {
   // * notice use this as a unsigned long
   private long parseUint64(DataInput cdi) {
```

Comment:
What is the difference btw this one and the one inside CDI?

Reply:
I am not sure. I prefer not to change the logic if we only do a cleanup.